src/Adapter/Twig/LocaleExtension.php

```diff
@@ -26,40 +26,11 @@
 
 namespace PrestaShop\PrestaShop\Adapter\Twig;
 
-use DateTime;
-use Twig\Extension\AbstractExtension;
-use Twig\TwigFunction;
+@trigger_error('This class is deprecated since 8.1 and will be dropped in 9.0.', E_USER_DEPRECATED);
 
 /**
  * Provides helper functions in Twig for formatting data using context locale
  */
-final class LocaleExtension extends AbstractExtension
+final class LocaleExtension extends \PrestaShopBundle\Twig\LocaleExtension
```

Comment:
Please always use `use` statements instead of FQCN in the code

```suggestion
use PrestaShopBundle\Twig\LocaleExtension as BaseLocaleExtension;

final class LocaleExtension extends BaseLocaleExtension
```